core/conversion/converters/impl/index_put.cpp

```diff
@@ -0,0 +1,144 @@
+#include <ATen/ATen.h>
```

Comment:
Can you just add the converter to shuffle.cpp or rename this scatter.cpp?

Reply:
renamed it to scatter.cpp